pages/docs/concepts/message.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Message
+weight: 2
+---
+
+## What is a message?
+`Message` is a method of communication used to transmit or exchange information from a sender to the receiver through the use of `channels`. A `message` is generic and can also be defined as an event, command, or request. The sender includes a payload that contains data that would be processed according to the preferred format defined by the receiver.
```

Comment:
```suggestion
`Message` is a method of communication used to transmit or exchange information from a sender to the receiver through the use of `channels`. A `message` is a generic term and can also be defined as an event or command. The sender includes a payload that contains data that would be processed according to the preferred format defined by the receiver.
```

I suggest we also write that a single message can be consumed by multiple independent receivers.

> A `message` is a generic term

not sure if I'm just a picky non-native speaker, but I think being so detailed is important here. 

> The sender includes a payload that contains data that would be processed according to the preferred format defined by the receiver.

I'm a bit confused about this sentence. Not sure what it means.

Reply:
So here all i'm saying is that the `sender` includes a payload that contains the information/data which is formatted according to the `receiver's` application e.g json, XML 

Hope that makes sense

Reply:
the thing is that receiver do not dictate how the message is formatted, so if sender sends json, then receiver much make sure that it can serialize it. This is what I think confuses me here

Reply:
Oooh yes, i get what you are saying now. Okay let me rephrase it properly then😅